ui/less/overflow_menu.less

```diff
@@ -34,8 +34,8 @@
   /* Where the menu appears. */
   position: absolute;
   z-index: 2;
-  right: 15px;
```

Comment:
Any specific reason we need to change the "right" property?

Reply:
the right property is set to static 15px, so I changed the right property to % based relative measurements, this way the right property adapts to the fullscreen as well as normal view.

Reply:
We'll need to double check how it looks on different sized screens (shrink and grow the browser window, and try device emulation in Chrome, and also on large screens like TV). If we have it in px, most likely means that it wasn't working with % for one of those use cases.

Reply:
I tested it on the browser in all possible screen sizes, also tried device emulation on chrome ( both small and large screen sizes ) and it works perfectly fine in all cases without any issue. 
for surety, you can do a check for these changes from your side.

Reply:
Thank you for testing!